build.bat

```diff
@@ -4,7 +4,19 @@ setlocal enabledelayedexpansion
 if "%DC%"=="" set DC="dmd"
 if "%DC%"=="ldc2" set DC="ldmd2"
 
-set DFLAGS=-O -release -inline -version=StdLoggerDisableWarning
+:: git might not be installed, so we provide 0.0.0 as a fallback or use
+:: the existing githash file if existent
+git describe --tags > githash_.txt
+for /f %%i in ("githash_.txt") do set githashsize=%%~zi
+if %githashsize% == 0 (
+	if not exist "githash.txt" (
+		echo "0.0.0" > githash.txt
```

Comment:
Just tested under wine, for some reason the printed version after compiling when git is not installed is `"0.0.0"` (with the quotes, instead of just `0.0.0`)